cloudinit/sources/DataSourceWSL.py

```diff
@@ -329,8 +329,12 @@ def _get_data(self) -> bool:
         # That's the reason for not using util.mergemanydict().
         merged: dict = {}
         overridden_keys: typing.List[str] = []
+        user_tags = None
```

Comment:
We can define this inline and drop the  set of this below. It'll end up None if no landscape.client.tags are defined. May have to check line length on this suggestion.
```suggestion
        user_tags = user_data.get("landscape", {}).get("client", {}).get("tags")
```